.github/workflows/release.yml

```diff
@@ -39,6 +39,7 @@ jobs:
         - cli-bin
         - cni-plugin
         - controller
+        - ns-labeler
```

Comment:
We could perhaps also build it separately in `integration.yaml`. I thought of its use in the integration tests. I agree it'll be helpful to have it in, but I'm not sure it's worth building every run...if we don't anticipate it to change often, we could just treat it the same way we do `proxy-init` and the `cni` plugin. The downside is, how do we actually trigger the tests to run _when we do_ run it.

I'm all for a separate release workflow. Perhaps (taking a page out of what I'm doing for testable release workflows), it would make since to have just one workflow where we test the job, and release. If we trigger the release through a label, we could hypothetically test everything first, and once tests are successful, re-trigger it upon labelling (which will force a release out). 

This will drag out the PR though so we could perhaps do it as a separate change where we just wire up CI? We could also run a small integration test for it, we should be able to do it if we keep it self-contained:

* Get a k3d cluster created, start a proxy through `kubectl`
* In the test: set-up a client, create namespace, create ConfigMap and all other prerequisites
* Trigger the labeler to run and check namespace meta at the end.

Should be low effort if we do it outside of the cluster, perhaps?

Reply:
We decided to finally host this in a separate repo given it's totally independent from the rest. It also makes sense from the CI perspective. Thanks for all the input, I'll ping you here when it's done :wink: